Resources/Prototypes/Entities/Objects/Specific/Security/evidence-marker.yml

```diff
@@ -0,0 +1,119 @@
+- type: entity
+  abstract: true
+  parent: BaseItem
+  id: EvidenceMarker
+  name: evidence marker
+  description: A numbered yellow marker, useful for labeling evidence on a crime scene.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Detective/evidence_marker.rsi
+  - type: Item
+    sprite: Objects/Specific/Detective/evidence_marker.rsi
+    size: Tiny
+    
+- type: entity
+  parent: EvidenceMarker
+  id: EvidenceMarkerOne
+  name: first evidence marker
```

Comment:
might make more sense to just have their names be `evidence marker` since it's already kinda clear what the number is. just a thought.